### PR TITLE
Create data models

### PR DESCRIPTION
**Pull request by anyepeter**
In this PR, I have created the data models with the following requirements 🔖 ;

### General requirements
✅ Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
✅ Make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
✅ Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
### Ruby requirements
✅ Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).
### Project requirements
✅ Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.

✅ Create a new app. Give it a meaningful name.
✅ Initialize your project with Git.
✅ Make sure that your project has Postgres database set up.
- Build your project schema.
   - Create and run the necessary migration files.
      ✅ Table and column names should match the ERD diagram.
      ✅ note: photo for users table should be a link to an image

✅ Foreign keys should be included.
✅ All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.text "Text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "PostId", null: false
+    t.bigint "AuthorId", null: false
+    t.index ["AuthorId"], name: "index_comments_on_AuthorId"
+    t.index ["PostId"], name: "index_comments_on_PostId"
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Let's try to follow the convention for naming table columns in Ruby. We follow snake_case, not camelCase. Kindly rename `Text` to `text`, `PostId` to `post_id`, `AuthorId` to `author_id`.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "AuthorId", null: false
+    t.bigint "PostId", null: false
+    t.index ["AuthorId"], name: "index_likes_on_AuthorId"
+    t.index ["PostId"], name: "index_likes_on_PostId"
+  end
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Let's try to follow the convention for naming table columns in Ruby. We follow snake case, not camel case. Kindly rename `PostId` to `post_id`, `AuthorId` to `author_id`.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  create_table "posts", force: :cascade do |t|
+    t.string "Title"
+    t.text "Text"
+    t.integer "CommentsCounter"
+    t.integer "LikesCounter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "AuthorId", null: false
+    t.index ["AuthorId"], name: "index_posts_on_AuthorId"
+  end
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Let's try to follow the convention for naming table columns in Ruby. We follow snake case, not camel case. Kindly rename `Title` to `title`; `Text` to `text`, `CommentsCounter` to `comments_counter`; `LikesCounter` to `likes_counter`; `AuthorId` to `author_id`.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @anyepeter,

Good job so far!
✔️ Use of PostgreSQL for DB
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @anyepeter 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented ✅ 
   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**